test/gurka/test_gtfs.cc

```diff
@@ -196,16 +198,48 @@ TEST(GtfsExample, WriteGtfs) {
       stopTwoID,
       stopThreeID + "_walk_the_plank_pal",
   };
-  for (int i = 0; i < 6; i++) {
-    struct StopTime stopTime {
-      .trip_id = "", .stop_id = "", .stop_sequence = 0, .arrival_time = Time("6:00:00"),
-      .departure_time = Time("6:00:00"), .stop_headsign = "head", .shape_dist_traveled = 0.0,
-      .timepoint = gtfs::StopTimePoint::Exact,
-    };
-    stopTime.stop_id = stopIds[i % 3];
-    stopTime.trip_id = (i < 3) ? tripOneID : tripTwoID;
-    feed.add_stop_time(stopTime);
-  }
+
+  struct StopTime trip_one_stop_one {
+    .trip_id = tripOneID, .stop_id = stopOneID + "_ledge_to_the_train_bucko", .stop_sequence = 0,
+    .arrival_time = Time("6:00:00"), .departure_time = Time("6:00:00"), .stop_headsign = "head",
+    .shape_dist_traveled = 0.0, .timepoint = gtfs::StopTimePoint::Exact,
+  };
+  feed.add_stop_time(trip_one_stop_one);
```

Comment:
I was being lazy doing this.. I create every stop_time by itself since it needs lots of kinda individual info it was lacking before, e.g. the `departure_time` actually needs to progress;)

Reply:
reminds me: check how the stop_time's `shape_dist_traveled` relates to the shapes.txt in valhalla